### PR TITLE
Update ember-cli to 2.3.0

### PR DESCRIPTION
**Pull request by hakubo**
After merging this everyone should run
`npm run clean-deep`
`npm run setup`

@rogatty @kvas-damian @adamkarminski

### CODE REVIEW

File: front/main/app/index.html [hunk 1]
```diff
@@ -49,9 +49,9 @@
 {{> qualaroo}}
 
 {{!--
-	Ember must render out to a specific div above the scripts, otherwise
-	Google PageSpeed will mistakenly believe that we have scripts loading and blocking content
-	DO NOT REMOVE THE FOLLOWING LINE:
+Ember must render out to a specific div above the scripts, otherwise
```
Comment (rogatty) [inline front/main/app/index.html]: Nope, let's keep it pretty.

File: front/main/app/resolver.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import Resolver from 'ember-resolver';
+
+export default Resolver;
```
Comment (rogatty) [inline front/main/app/resolver.js]: Why is that better?

Comment (hakubo) [inline front/main/app/resolver.js]: talk to Ember CLI maintainers :)

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "clean": "rm -rf www/ node_modules/ front/auth/bower_components/ front/common/bower_components/ front/main/bower_components/ front/main/node_modules/ server/node_modules/",
     "clean-deep": "npm run clean && npm cache clean && bower cache clean",
     "setup": "npm run setup-root && npm run setup-common && npm run setup-auth && npm run setup-main && npm run setup-server",
-    "setup-for-local": "npm install -g bower gulp@3.x.x ember-cli@2.2.0-beta.6",
+    "setup-for-local": "npm install -g bower gulp@3.x.x ember-cli@2.3.0",
```
Comment (rogatty) [inline package.json]: We need to update chef-repo too.

**Comment by rogatty** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**